docs/sgp/sgp_config_options.md

```diff
@@ -0,0 +1,27 @@
+# Configuration options for Security groups for pods
+
+Users are able to configure the controller functionality related to security group for pods by updating the `data` fields in EKS-managed configmap `amazon-vpc-cni`.
+
+* **branch-eni-cooldown**: Cooldown period for the branch ENIs, the period of time to wait before deleting the branch ENI for propagation of iptables rules for the deleted pod. The default cooldown period is 60s, and the minimum value for the cool period is 30s. If user updates configmap to a lower value than 30s, this will be overridden and set to 30s.
+
+Minimal configmap example:
+```
+apiVersion: v1
+data:
+  branch-eni-cooldown: "60"
+kind: ConfigMap
+metadata:
+  name: amazon-vpc-cni
+  namespace: kube-system
+```
+
+Note that if you set the cooldown period to be lower than the default, you would also need to adjust the kernel's ARP cache timeout to be equal to the new cooldown period. Else, this will result in intermittent DNS timeouts or packet drops due to re-use of the IP address of a recently terminated pod by a new pod before the kernel's ARP cache can be updated.
+
+Customers may use the below commands to tune the ARP cache timeout, make sure you update it for all interfaces on the node. 
```

Comment:
default will auto update the new interfaces. can we make this more clear? Thanks.